_topics/en/transaction-origin-privacy.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Transaction origin privacy
+
+## Optional.  Shorter name to use for reference style links e.g., "foo"
+## will allow using the link [topic foo][].  Not case sensitive
+# shortname: foo
+
+## Optional.  An entry will be added to the topics index for each alias
+#aliases:
+#  - Foo
+
+## Required.  At least one category to which this topic belongs.  See
+## schema for options
+categories:
+  - Transaction Relay Policy
+  - Privacy Enhancements
+  - Privacy Problems
+
+## Optional.  Produces a Markdown link with either "[title][]" or
+## "[title](link)"
+primary_sources:
+    - title: "Bitcoin Wiki: Privacy---Network analysis"
+      link: https://en.bitcoin.it/wiki/Privacy#Traffic_analysis
+
+## Optional.  Each entry requires "title" and "url".  May also use "feature:
+## true" to bold entry and "date"
+optech_mentions:
+  - title: "Bitcoin Core #18861 won't reply to with txes it hasn't yet announced to the requesting peer"
+    url: /en/newsletters/2020/05/27/#bitcoin-core-18861
+
+  - title: "Transcript: TxProbe: discovering Bitcoin's network topology using orphan transactions"
+    url: /en/newsletters/2019/09/18/#txprobe-discovering-bitcoin-s-network-topology-using-orphan-transactions
+
+  - title: "Bitcoin Core #14897 introduces a semi-random order when requesting transactions"
+    url: /en/newsletters/2019/02/12/#bitcoin-core-14897
```

Comment:
We did mention both of those in subsequent newsletters:

- https://bitcoinops.org/en/newsletters/2019/04/23/#bitcoin-core-15839
- https://bitcoinops.org/en/newsletters/2019/06/19/#bitcoin-core-15834

I've added 15834 to the topic as it also links to 15839 (I don't want to add both of them since they're not directly about transaction origin privacy; I do want to add one of them, though, as it's conceivable that another implementation could stumble upon the same problem).